package.json

```diff
@@ -33,6 +33,12 @@
   "browser": {
     "./metasync.js": "./dist/metasync.js"
   },
+  "files": [
+    ".babelrc",
+    "dist/",
+    "lib/",
+    "metasync.js"
```

Comment:
It's better to remove this and fix the `main` field to point correctly to `metasync.js`, not `./metasync.js`.
```suggestion
```